addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/DirectoryBrowsingScanRule.java

```diff
@@ -105,21 +110,31 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         }
         if (evidence != null && evidence.length() > 0) {
             // we found something
-            newAlert()
-                    .setName(getName() + " - " + server)
-                    .setRisk(Alert.RISK_MEDIUM)
-                    .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                    .setDescription(getDescription() + " - " + server)
-                    .setOtherInfo(getExtraInfo(msg, evidence))
-                    .setSolution(getSolution())
-                    .setReference(getReference())
-                    .setEvidence(evidence)
-                    .setCweId(548) // Information Exposure Through Directory Listing
-                    .setWascId(16) // Directory Indexing
-                    .raise();
+            this.buildAlert(server, evidence, msg).raise();
         }
     }
 
+    private AlertBuilder buildAlert(String server, String evidence, HttpMessage msg){
+        return newAlert()
+                .setName(getName() + " - " + server)
+                .setRisk(Alert.RISK_MEDIUM)
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setDescription(getDescription() + " - " + server)
+                .setOtherInfo(getExtraInfo(msg, evidence))
+                .setSolution(getSolution())
+                .setReference(getReference())
+                .setEvidence(evidence)
+                .setCweId(548) // Information Exposure Through Directory Listing
+                .setWascId(16); // Directory Indexing
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+        alerts.add(buildAlert(server,evidence,msg).build());
+        return alerts;
```

Comment:
I believe this could just return `List.of(buildAlert(server, evidence, msg).build());`

Reply:
My bad, I think I misunderstood what the purpose  is of getExampleAlerts(). I'm just not that sure what good examples would be, because after looking around it never has something consistent I'm sure of that I can use.